CHANGELOG.md

```diff
@@ -10,6 +10,9 @@ Changes
 ### Details
 https://github.com/DataDog/dd-agent/compare/5.27.0...5.27.1
 
+### Known Issues
+The dogstream fix introduced in [#3701][] will result in a nasty file descriptor leak. If you use dogstream please downgrade to 5.26.0.
```

Comment:
suggestions:
* `the dogstream fix` -> `the dogstream change`
* `will result in a nasty file descriptor leak` -> `causes a significant file descriptor leak`